master/internal/sproto/task_actor.go

```diff
@@ -45,6 +47,12 @@ type (
 		ContainerStopped *TaskContainerStopped
 	}
 
+	// UpdatePodStatus notifies the resource manager of experiment state change.
```

Comment:
nit: "experiment"? maybe job or task would apply better?

Reply:
This is only applicable to kubernetes though, would it be accurate to associate it with experiments in general?

Reply:
we chatted about this offline